pandas/tests/indexes/datetimes/test_setops.py

```diff
@@ -471,7 +471,7 @@ def test_intersection_bug(self):
     def test_intersection_list(self):
         # GH#35876
         values = [pd.Timestamp("2020-01-01"), pd.Timestamp("2020-02-01")]
-        idx = pd.DatetimeIndex(values, name="a")
+        idx = pd.DatetimeIndex(values)
         res = idx.intersection(values)
         tm.assert_index_equal(res, idx)
```

Comment:
could keep the name in `idx` and test for `tm.assert_index_equal(res, idx.rename(None))`?